wirecard/oxid-ee
# WDPD-61: Add new languages

### This PR

* Adds translations for the plugin (fr, id, jp, ko, zh_CN, zh_TW)

### Notes

* PR has much more than 400 loc, because of the added language files
* Strings without a translation for a specific language will be shown in english
* For Korean(kr) no translation files are provided by oxid yet

### TODO

* Testing 

### How to Test

* Make sure the needed translation files are copied to the corresponding shop directories(https://oxidforge.org/en/language-handling.html)
* Add new languages, check if all strings are translated as expected

### Jira Links
https://jira.parkside.at/browse/WDPD-61
